### DISCUSSION TITLE
Simple Problem

### DISCUSSION THREAD

**Discussion by SoftPlay0** [lukas-blecher/LaTeX-OCR#209] [Q&A]
Hello everyone, probably my problem is very simple but not being practical of pyton I find it difficult to solve. 
I installed pyton and successively pytorch  and pix2tex. When I try to run the latexocr command I get this feedback any idea?

PS D: Users \Name \Downloads \LaTeX-OCR-main> latexocr
latexocr : Term 'latexocr' not recognized as cmdlet name, function, executable program, or script file. Check the spelling of the 
name or verify that the path is included and correct, then try again.
In line:1 car:1
+ latexocr
+ ~~~~~~~
    + CategoryInfo   : ObjectNotFound: (latexocr:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by lukas-blecher** [comment]
is python and co in your PATH?
check if you can call just `python`. If not, make sure to add it to your PATH.

**Comment by SoftPlay0** [comment]
I verified that python is in the folder by getting the following output.
Since the problem persists I'll explain the steps I performed during the installation:
1- I installed pyton 3.10 by downloading the app from the windows store
2- I downloaded the python package for visual studio code
3-I opened your repository in the Visual Studio code
4- I opened a new terminal and ran the pytorch installation command:
pip3 install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu113
5- I ran the command: 
pip install pix2tex[gui]

Are there any mistakes?

**Reply by lukas-blecher** [reply]
Sounds like it should work. Steps 2 and 3 are not really needed.

Did any of the installations fail? What happens if you open a new terminal and just run `python` and then enter
```python
import torch
import pix2tex
```
If no errors occur maybe the python `Scripts` directory is not in your PATH.

**Reply by SoftPlay0** [reply]
During the installation I did not receive Error Messages.
I ran the commands you suggested and got this output:
PS D: Username\ Download\ LaTeX-OCR-main> python
Python 3.10.8 (tag/v3.10.8:aaaf517, 11 October 2022, 16:50:30) [MSC v.1933 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import the flashlight
>>> import pix2tex
>>> pix2tex
Modulo 'pix2tex' da’D:  Users\\name\\Download\ \LaTeX-OCR-main \\pix2tex\\ _init__.py'>
>>>
If the problem is that the python Scripts directory is not in mine PATH, how can i fix it?

**Reply by lukas-blecher** [reply]
You could try
```
python -m pix2tex.__main__ --gui
```
That should work even without scripts in PATH.

**Reply by SoftPlay0** [reply]
With this command your script work perfectly also with very long formulas.
I don not understand how can i put python in the sript but don't worry for this.
Thank you a lot  for your availability!

**Comment by SoftPlay0** [comment]
If you need more information about the visual studio code output don't exitate to ask me
